icons/construction.svg

```diff
@@ -9,12 +9,12 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M21 6H3a1 1 0 0 0-1 1v6a1 1 0 0 0 1 1h18a1 1 0 0 0 1-1V7a1 1 0 0 0-1-1Z" />
-  <path d="M17 14v7" />
-  <path d="M7 14v7" />
-  <path d="M17 3v3" />
-  <path d="M7 3v3" />
-  <path d="M2.5 6.5 10 14" />
-  <path d="m8 6 8 8" />
-  <path d="m14 6 7.5 7.5" />
+  <rect x="2" y="6" width="20" height="8" rx="1"/>
+  <path d="M17 14v7"/>
+  <path d="M7 14v7"/>
+  <path d="M17 3v3"/>
+  <path d="M7 3v3"/>
+  <path d="M2.28 6.3 10 14"/>
+  <path d="m14 6 7.7 7.73"/>
+  <path d="m8 6 8 8"/>
```

Comment:
Nice catch, but there's a slight path mismatch in the bottom right corner:
```suggestion
  <rect x="2" y="6" width="20" height="8" rx="1" />
  <path d="M17 14v7" />
  <path d="M7 14v7" />
  <path d="M17 3v3" />
  <path d="M7 3v3" />
  <path d="M10 14 2.3 6.3" />
  <path d="m14 6 7.7 7.7" />
  <path d="m8 6 8 8"/>
```
(And again, we currently need the spaces, or the linter will change the files)